tools/cldr-apps/js/src/esm/cldrNotify.mjs

```diff
@@ -51,33 +51,51 @@ function open(message, description, onClick) {
   });
 }
 
+function logInfo(message, description) {
+  if (hasDataDog) {
+    datadogLogs.logger.info(message, { description });
+  } else {
+    console.log(message);
+  }
+}
+
 /**
  * Display a warning notification
  *
  * @param {String} message the title, displayed at the top
  * @param {String} description the more detailed description
  */
 function warning(message, description) {
-  if (hasDataDog) {
-    datadogLogs.logger.warn(message, { description });
-  }
+  logWarning(message, description);
   notification.warning({
     message: message,
     description: description,
     duration: MEDIUM_DURATION,
   });
 }
 
+/**
+ * Log a warning
+ *
+ * @param {String} message the title
+ * @param {String} description the more detailed description
+ */
+function logWarning(message, description) {
+  if (hasDataDog) {
+    datadogLogs.logger.warn(message, { description });
+  } else {
+    console.warn(message);
+  }
```

Comment:
When we're testing on production or cldr-smoke or cldr-staging, it might still be helpful if warnings/errors would show up in the browser console, even if they also show up in datadog